maps/cam_lao_nam/mission.sqm

```diff
@@ -41073,13 +41075,83 @@ class Mission
 								nAttributes=2;
 							};
 						};
+						class Item64
+						{
+							dataType="Object";
+							class PositionInfo
+							{
+								position[]={10923.218,70.040001,14374.318};
+							};
+							side="Empty";
+							class Attributes
+							{
+							};
+							id=18737;
+							type="Land_HelipadEmpty_F";
+						};
+						class Item65
+						{
+							dataType="Object";
+							class PositionInfo
+							{
+								position[]={11283.754,69.998314,14374.168};
+							};
+							side="Empty";
+							class Attributes
+							{
+							};
+							id=18738;
+							type="Land_HelipadEmpty_F";
+						};
+						class Item66
+						{
+							dataType="Object";
+							class PositionInfo
+							{
+								position[]={11395.205,70.089996,14374.39};
+							};
+							side="Empty";
+							class Attributes
+							{
+							};
+							id=18739;
+							type="Land_HelipadEmpty_F";
+						};
+						class Item67
+						{
+							dataType="Object";
+							class PositionInfo
+							{
+								position[]={11700.369,69.910004,14373.915};
+							};
+							side="Empty";
+							class Attributes
+							{
+							};
+							id=18740;
+							type="Land_HelipadEmpty_F";
+						};
+						class Item68
+						{
+							dataType="Object";
+							class PositionInfo
+							{
+								position[]={11926.36,70.261833,14373.789};
+							};
+							side="Empty";
+							class Attributes
+							{
+							};
+							id=18741;
+							type="Land_HelipadEmpty_F";
+						};
```

Comment:
not fixed the gia lam issue 😢 

not sure what the deal with this is then as everything is attached and placed correctly

might need further/deeper script investigation